subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/NebulaPluginsSmokeTest.groovy

```diff
@@ -23,7 +23,7 @@ class NebulaPluginsSmokeTest extends AbstractSmokeTest {
         buildFile << """
             plugins {
                 id "java"
-                id "nebula.dependency-recommender" version "3.3.0"
+                id "nebula.dependency-recommender" version "4.1.2"
             }
```

Comment:
There're also a new versions for other Nebula plugins:

- `nebula.plugin-plugin` -> 5.6.0: https://plugins.gradle.org/plugin/nebula.plugin-plugin
- `nebula.lint` -> 7.3.5: https://plugins.gradle.org/plugin/nebula.lint
- `nebula.dependency-lock` -> 4.9.4:https://plugins.gradle.org/plugin/nebula.dependency-lock